code/clue.js

```diff
@@ -10,50 +9,202 @@ const mrGreen = {
+  weight: 14,
+  color: 'black',
+  image: 'assets/dumbbell.png'
+}
+
+const poison = {
+  name: 'poison',
+  weight: 1,
+  color: 'black',
+  image: 'assets/poison.png'
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const axe = {
+  name: 'axe',
+  weight: 12,
+  color: 'black',
+  image: 'assets/axe.png'
+}
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const bat = {
+  name: 'bat',
+  weight: 9,
+  color: 'black',
+  image: 'assets/bat.png'
+}
+
+const trophy = {
+  name: 'trophy',
+  weight: 7,
+  color: 'black',
+  image: 'assets/trophy.png'
+}
+
+const pistol = {
+  name: 'pistol',
+  weight: 5,
+  color: 'black',
+  image: 'assets/pistol.png'
+}
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
+  mrsWhite,
+  mrsPeacock,
+  colonelMustard,
+  professorPlum,
+  missScarlet
+]
+
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
 ]
 
-const weapons = []
+const rooms = [
+  'Dining Room',
+  'Conservatory',
+  'Kitchen',
+  'Study',
+  'Library',
+  'Billiard Room',
+  'Lounge',
+  'Ballroom',
+  'Hall',
+  'Spa',
+  'Living Room',
+  'Observatory',
+  'Theater',
+  'Guest House',
+  'Patio'
+]
+
+console.log(suspects)
+console.log(weapons)
+console.log(rooms)
 
-const rooms = []
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
-  return array[Math.floor(Math.random() * array.length)]
+  const randomArrayIndex = Math.floor(Math.random() * array.length)
+  return array[randomArrayIndex]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+const mystery = {
+  killer: undefined,
+  weapon: undefined,
+  room: undefined
+}
 
-// This function will be invoked when you click on the killer card.
 const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
   mystery.killer = randomSelector(suspects)
-
-  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. Feel free to add more things to show about the killer.
   document.getElementById('killerCard').style.background = mystery.killer.color
-  document.getElementById(
-    'killerName'
-  ).innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerName').innerHTML = `${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerAge').innerHTML = `${mystery.killer.age}`
+  document.getElementById('killerOccupation').innerHTML = `${mystery.killer.occupation}`
+  document.getElementById('killerImage').src = mystery.killer.image
+  document.getElementById('killerDescription').innerHTML = `${mystery.killer.description}`
```

Comment:
If you're only going to write JavaScript (not concatenating a string like
```
`My name is ${name}`
```
you don't need the template literals, but can simply write pure JavaScript like:
`document.getElementById('killerAge').innerHTML = mystery.killer.age`